### PR TITLE
feat: Ad css style for people advanced filter drawer - EXO-60555 - Meeds-io/MIPs#39

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less [hunk 1]
```diff
@@ -53,6 +53,16 @@
   }
 }
 
+#resetButtonFilterPeopl {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]:
```suggestion
.peopleAdvancedFilterResetButton {
```
try to move it directly under #peopleListApplication

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less (hunk 1 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]: fixed !

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less [hunk 2]
```diff
@@ -8,6 +8,15 @@
     width: auto;
     padding-right: 20px;
   }
+  .peopleAdvancedFilterResetButton {
+    cursor: pointer!important;
+    color: var(--allPagesPrimaryColor, #476a9c) !important;
+    box-shadow: 0 0 #FFFFFE!important;
+    background-color: #FFFFFF!important;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]: we should try to use color codes here

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less (hunk 2 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]: fixed !

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less [hunk 3]
```diff
@@ -8,6 +8,15 @@
     width: auto;
     padding-right: 20px;
   }
+  .peopleAdvancedFilterResetButton {
+    cursor: pointer!important;
+    color: var(--allPagesPrimaryColor, @primaryColorDefault) !important;
+    box-shadow: 0 0 @baseBackgroundDefault!important;
+    background-color: @baseBackgroundDefault!important;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]:
```suggestion
    box-shadow: 0 0 @baseBackgroundDefault !important;
    background-color: @baseBackgroundDefault !important;
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less (hunk 3 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/PeopleList/Style.less]: fixed !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**